Added reference image to export docs (#431)
pupil-labs/pupil-docs: src/cloud/enrichments/README.md

| **gaze position in reference image y [px]** | Same as "gaze position in reference image x [px]" but for the y-coordinate.     |

#### Reference Image

> The reference image used for defining the enrichment. It has the same name and file extension as the original file.

Reviewer: The file name will not be the same. Please see suggestion below. 

```suggestion
The reference image used for defining the enrichment. The file is named `reference_image.jpeg|png`
```
